cheroot/server.py

```diff
@@ -340,8 +341,65 @@ def __next__(self):
 
     next = __next__
 
+class RFile:
+    def __init__(self, rfile):
+        self.rfile = rfile
+
+    def read(self, size=None):
```

Comment:
Could you please use ABCMeta and mark this as an `@abstractmethod`? (and all the other methods too, of course)